graylog2-web-interface/src/components/common/Select.jsx

```diff
@@ -135,8 +129,6 @@ const Select = createReactClass({
   },
 
   _select: undefined,
-  reactSelectStyles: require('!style/useable!css!react-select/dist/react-select.css'),
```

Comment:
Instead of removing this and not using the (cleaner) way of `use()/unuse()`, we should check if it is possible to mock the `require` here in the test. Just returning a dummy object with the accessed fields/functions should be sufficient.